content/en/Integrations/IntegrationBuilder/configure-gitlab-integration.md

```diff
@@ -0,0 +1,138 @@
+
+{{% image src="/integrations/integration_builder/gitlab/05-gitlab-connection-data.png" alt="Create GitLab connection" %}}
+
+Alternatively, you can use password-based authentication. In this case, you need to provide your GitLab username and password.
+
+{{% image src="/integrations/integration_builder/gitlab/06-gitlab-connection-auth-type.png" alt="GitLab auth types" %}}
+
+Once done, click "Connect".
+
+{{% image src="/integrations/integration_builder/gitlab/08-connection-connected.png" alt="GitLab connection successful" %}}
+
+### 4. Create a New Folder to Better Organize Your Recipes
+
+Before creating a new recipe, it is better to create a new folder for all future GitLab recipes.
+
+Go to the project page again, and click on the "Create" button on the top right. This time, choose "Folder".
+
+{{% image src="/integrations/integration_builder/gitlab/09-new-folder.png" alt="Create a new folder" %}}
+
+Give the new folder a name, and choose its location. Then click "Create Folder".
+
+{{% image src="/integrations/integration_builder/gitlab/10-new-folder-data.png" alt="New folder data" %}}
+
+Now you can see the new folder in the project.
+
+{{% image src="/integrations/integration_builder/gitlab/11-folder-created.png" alt="New folder created" %}}
+
+Now we have the option in the folder to create a new recipe. But it is better to reuse an existing recipe.
+
+### 5. Find an Existing Recipe
+
+Go to the "Library" tab and search for the GitLab recipe.
+
+{{% image src="/integrations/integration_builder/gitlab/12-find-community-recipe.png" alt="Find GitLab recipe" %}}
+
+There is no GitLab recipe found for a new pentest finding, but a recipe for a DAST finding exists. We can use it as a base and customize it. Click on the recipe to see its details and click "Use this recipe".
+
+{{% image src="/integrations/integration_builder/gitlab/13-use-recipe.png" alt="DAST recipe" %}}
+
+Choose to save the recipe in the folder you created earlier.
+
+{{% image src="/integrations/integration_builder/gitlab/14-use-recipe-folder.png" alt="Save recipe in folder" %}}
+
+Once copied and saved, an initial customization is needed.
+
+{{% image src="/integrations/integration_builder/gitlab/15-recipe-needs-customization.png" alt="Customize recipe" %}}
+
+You can see in the recipe's graph that there is an error. The highlighted step is the one causing the error.
+
+{{% image src="/integrations/integration_builder/gitlab/16-error.png" alt="Recipe error" %}}
+
+The existing error is due to a missing field for the GitLab project.
+
+{{% image src="/integrations/integration_builder/gitlab/17-click-step-with-error.png" alt="Recipe error" %}}
+
+Select your GitLab project in the step's settings.
+
+{{% image src="/integrations/integration_builder/gitlab/18-select-gitlab-project.png" alt="Select GitLab project" %}}
+
+### 6. Customize the Recipe
+
+Notice the first step in the recipe (the trigger) is about a DAST finding. We need to change it to a pentest finding.
```

Comment:
If I understand correctly we copied a DAST-focused recipe to turn it into a pentest-focused one. 
This might be confusing to users, since there's already a recipe they can pick to cover pentest findings. I recommend rather choosing the pentest recipe from the beginning, in order to avoid having to do major adjustments.

Reply:
The recipe name for the pentest finding looked confusing to me. It looked like a state transition recipe. 
Now I understand what it does. Still confusing to any customer without our documentation.

Reply:
Yes, I agree we should also consider renaming either recipe to avoid confusion. If we're having issues, our customers will too.